### PR TITLE
Fixes #8 - lint errors

### LINKED ISSUE #8: Fix lint

### PR DESCRIPTION
**Pull request by ugocupcic**

### CODE REVIEW

File: sr_example/package.xml [hunk 1]
```diff
@@ -1,34 +1,36 @@
 <package>
-  <name>sr_example</name>
-  <version>1.4.0</version>
-  <description>sr_example is an example of a real package interfaced with our robot. Please refer to the tutorial Creating a package to interact with our robots.</description>
-  <maintainer email="software@shadowrobot.com">Shadow Robot's software team</maintainer>
+    <name>sr_example</name>
```
Comment (VahidAminZ) [inline sr_example/package.xml]: Is there a reason for indentation of xml files with 4 spaces?

File: sr_hand_kinematics/hand_kinematics/include/hand_kinematics/hand_kinematics_plugin.h [hunk 2]
```diff
@@ -66,10 +66,10 @@
 
 namespace hand_kinematics
 {
-class HandKinematicsPlugin : public kinematics::KinematicsBase
+  class HandKinematicsPlugin :
```
Comment (VahidAminZ) [inline sr_hand_kinematics/hand_kinematics/include/hand_kinematics/hand_kinematics_plugin.h:69]: content of the namespace should not be indented.

File: sr_example/package.xml (hunk 1 above)
Comment (ugocupcic) [inline sr_example/package.xml]: not really, clion's automatic formatting on all files did that.

File: sr_hand_kinematics/hand_kinematics/include/hand_kinematics/hand_kinematics_plugin.h (hunk 2 above)
Comment (ugocupcic) [inline sr_hand_kinematics/hand_kinematics/include/hand_kinematics/hand_kinematics_plugin.h:69]: I can't find the option to do that in the clion formatter + roslint is not sending an error :)

File: sr_example/package.xml (hunk 1 above)
Comment (ugocupcic) [inline sr_example/package.xml]: OK I fixed it to use 2 spaces

**Pull request merged**